jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/test/java/org/jbpm/workbench/ht/client/editors/taskslist/AbstractTaskListViewTest.java

```diff
@@ -288,6 +284,21 @@ public void testSuspendActionHasCell(){
         }
     }
     
+    @Test
+    public void testViewProcessActionHasCell(){
+        ConditionalActionHasCell viewProcCell = new ConditionalActionHasCell(
+                "",
+                cellDelegate,
+                task -> (task.getProcessInstanceId() >= 0)
+        );
+        
+        TaskSummary testTask = createTestTaskSummary(TASK_STATUS_READY, TEST_USER_ID, POSITIVE_POTENTIAL_OWNERS);
```

Comment:
@bsremac status should not be a problem as the data still available even if process/task is completed/aborted. The only condition is to have access to the perspective.